src/components/structures/BackdropPanel.tsx

```diff
@@ -0,0 +1,167 @@
+/*
+Copyright 2021 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { createRef } from "react";
+import "context-filter-polyfill";
+
+import UIStore from "../../stores/UIStore";
+
+interface IProps {
+    backgroundImage?: CanvasImageSource;
+}
+
+interface IState {
+    // Left Panel image
+    lpImage?: string;
+    // Left-left panel image
+    llpImage?: string;
+}
+
+export default class BackdropPanel extends React.PureComponent<IProps, IState> {
+    private leftLeftPanelRef = createRef<HTMLCanvasElement>();
+    private leftPanelRef = createRef<HTMLCanvasElement>();
+
+    private sizes = {
+        leftLeftPanelWidth: 0,
+        leftPanelWidth: 0,
+        height: 0,
+    };
+    private style = getComputedStyle(document.documentElement);
+
+    constructor(props: IProps) {
+        super(props);
+        this.state = {};
+    }
+
+    public componentDidMount() {
+        UIStore.instance.on("SpacePanel", this.onResize);
+        UIStore.instance.on("GroupFilterPanelContainer", this.onResize);
+        this.onResize();
+    }
+
+    public componentWillUnmount() {
+        UIStore.instance.off("SpacePanel", this.onResize);
```

Comment:
Should there be a matching one for `GroupFilterPanelContainer`?

Reply:
`GroupFilterPanelContainer` can't be resized, can it?

Reply:
Definitely, yes! Forgot about that one, thanks for catching!

Reply:
@SimonBrandner It can't, but just for the sake of normalization, if it will be resizable any time in the future it'll still work.

Reply:
> if it will be resizable any time in the future it'll still work.

I doubt we will make it resizable before we kill it with the rest of communities :P